[FW][FIX] account_withholding_automatic: problema de release con loc española

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/account-payment#547

pablohmontenegro committed:
[FIX] account_withholding_automatic: problema de release con loc española

Ticket: 81960
Va de la mano con este pr https://github.com/ingadhoc/odoo-argentina/pull/934

X-original-commit: 5ad7a7621b97c26d8fdfd3245c6d9fd2265bad32

roboadhoc commented:
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/account-payment/pull/550).

fw-bot-adhoc commented:
@pablohmontenegro @zaoral cherrypicking of pull request ingadhoc/account-payment#547 failed.

stdout:
```
CONFLICT (modify/delete): account_withholding_automatic/models/account_tax.py deleted in HEAD and modified in 410bcbf6 ([FIX] account_withholding_automatic: problema de release con loc española). Version 410bcbf6 ([FIX] account_withholding_automatic: problema de release con loc española) of account_withholding_automatic/models/account_tax.py left in tree.
CONFLICT (modify/delete): account_withholding_automatic/__manifest__.py deleted in HEAD and modified in 410bcbf6 ([FIX] account_withholding_automatic: problema de release con loc española). Version 410bcbf6 ([FIX] account_withholding_automatic: problema de release con loc española) of account_withholding_automatic/__manifest__.py left in tree.

```

stderr:
```
19:16:59.385781 git.c:444               trace: built-in: git cherry-pick 410bcbf6914e6eabe1235dffbf3ee48d9c090c28
error: could not apply 410bcbf6... [FIX] account_withholding_automatic: problema de release con loc española
hint: after resolving the conflicts, mark the corrected paths
hint: with 'git add <paths>' or 'git rm <paths>'
hint: and commit the result with 'git commit'
----------
status:

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port

zaoral commented:
we delete this FW because the module has been deprecated in 17+
